### ISSUE TITLE
Can't build the image on Docker Hub

### ISSUE THREAD

**Issue by Hetsh** [Hetsh/docker-starbound#1]
Dedicated server can't be downloaded without logging into a valid steam account and there is no easy way to hide the credentials.

**Comment by spannerman79** [comment]
Yes it is stupid that your creds are needed to download a headless server.  Game Dev's don't understand how servers actually work.
